pkg/kubenest/util/helper.go

```diff
@@ -214,3 +221,102 @@ func DeleteObject(dynamicClient dynamic.Interface, namespace string, name string
 	}
 	return nil
 }
+
+// DecodeYAML unmarshals a YAML document or multidoc YAML as unstructured
+// objects, placing each decoded object into a channel.
+// code from https://github.com/kubernetes/client-go/issues/216
+func DecodeYAML(data []byte) (<-chan *unstructured.Unstructured, <-chan error) {
```

Comment:
todo

Reply:
anp containers use apply to add to apiserver deployment